### PR TITLE
[Reviewer: Andy] Etcd plugins now receive parameter objects

### PR DESCRIPTION
**Pull request by AlexHockey**
Co-req change to accompany Metaswitch/clearwater-etcd#70

### CODE REVIEW

File: sprout-base.root/usr/share/clearwater/clearwater-config-manager/plugins/sprout_scscf_json_plugin.py [hunk 1]
```diff
@@ -44,8 +44,8 @@
 _log = logging.getLogger("sprout_scscf_json_plugin")
 
 class SproutSCSCFJsonPlugin(SproutJsonPlugin):
-    def __init__(self):
+    def __init__(self, _params):
```
Comment (bossmc) [inline sprout-base.root/usr/share/clearwater/clearwater-config-manager/plugins/sprout_scscf_json_plugin.py]: Is this a convention I'm not aware of?  Does the underscore indicate "unused"?

**Comment by bossmc** [comment]
One comment (that applies throughout and probably applies to the other PRs).

File: sprout-base.root/usr/share/clearwater/clearwater-config-manager/plugins/sprout_scscf_json_plugin.py (hunk 1 above)
Comment (AlexHockey) [inline sprout-base.root/usr/share/clearwater/clearwater-config-manager/plugins/sprout_scscf_json_plugin.py]: Other repos were already doing this so I did the same thing here. It doesn't match a convention I know about but I quite like it :-)

Generally in python you normally denote an unused variable with an underscore. This is  fine for tuple unpacking (e.g. `host, _, _ = get_connection_info()` ignores the protocol and port that the hypothetical function returns). It doesn't work well for function arguments - you can't use two underscores as you will be using duplicate argument names.

Comment (bossmc) [inline sprout-base.root/usr/share/clearwater/clearwater-config-manager/plugins/sprout_scscf_json_plugin.py]: Ok, happy with that.

**Pull request merged**